src/Our.Umbraco.HttpsRedirect/Properties/AssemblyInfo.cs

```diff
@@ -12,3 +12,6 @@
 
 [assembly: ComVisible(false)]
 [assembly: Guid("066DE8E5-30D8-4911-A53D-2F21A24ADD4D")]
+[assembly: AssemblyVersion("1.3.0.0")]

```

Comment:
The MSBuild script we use to create the package will automatically add/update the `AssemblyVersion` number to the "VersionInfo.cs" file.  Adding it back in to "AssemblyInfo.cs" would break the build.